src/main/java/com/spotify/docker/client/messages/AttachedNetwork.java

```diff
@@ -27,14 +27,20 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.auto.value.AutoValue;
-
+import com.google.common.collect.ImmutableList;
+import java.util.List;
 import javax.annotation.Nullable;
 
+
 @AutoValue
 @JsonAutoDetect(fieldVisibility = ANY, getterVisibility = NONE, setterVisibility = NONE)
 public abstract class AttachedNetwork {
 
   @Nullable
+  @JsonProperty("Aliases")
+  public abstract ImmutableList<String> aliases();
```

Comment:
I don't think so. 
Actually, I don't think aliases are ever null when supported by the API (at least the container id).
For retrocompatibility purposes, I put it as nullable so that it doesn't break for earlier versions.